postgres-appliance/basebackup.sh

```diff
@@ -37,7 +37,7 @@ function receivexlog() {
         kill -0 $receivexlog_pid && sleep 1 || exit
     done
 
-    kill $receivexlog_pid
+    kill $receivexlog_pid && rm -fr ${XLOG_FAST}/*
```

Comment:
I would actually give a time to pg_receivexlog to exit, lets say 1 second.
And we don't need to remove recursively, `rm -f ${XLOG_FAST}/*` should be enough.